benchmarks/startup/scenario.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+
+import bm
+
+
+class Startup(bm.Scenario):
+    telemetry_enabled = bm.var_bool()
+    use_ddtrace_run = bm.var_bool()
+    with_trace = bm.var_bool()
+
+    def run(self):
+        cmd = "python"
+        if self.use_ddtrace_run:
+            cmd = "ddtrace-run python"
+
+        code = "from ddtrace import tracer;"
+        if self.with_trace:
+            code += "with tracer.trace('test'): pass;"
+
+        def _(loops):
+            for _ in range(loops):
+                subp = subprocess.Popen(
+                    cmd, "-c", code, env=self.get_env(), stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True
```

Comment:
you might want to generate the env dict once outside `def _(loops):` to cut down on the function calls in the measured function.

... probably doesn't matter _that_ much, so just an observation/nit not a necessary change.